src/commands/protocols/get/sender/local-get-command.js

```diff
@@ -104,7 +101,7 @@ class LocalGetCommand extends Command {
                 contract,
                 knowledgeCollectionId,
                 knowledgeAssetId,
-                TRIPLES_VISIBILITY.ALL,
+                contentType,
```

Comment:
If user didn't send it, set it up on controller to be TRIPLES_VISIBILITY.ALL by default